api/layer/tree_mock.go

```diff
@@ -1,8 +1,7 @@
-package tree
+package layer
```

Comment:
What the idea behind moving it into `layer` package? `neofs_mock.go` is still in `neofstest` package meanwhile.

Reply:
Cyclic dependency. Like https://github.com/nspcc-dev/neofs-s3-gw/issues/438